src/main/java/gov/nasa/jpl/mbee/mdk/MDKConfigurator.java

```diff
@@ -141,9 +141,6 @@ private void addElementActions(ActionsManager manager, Element e, List<Element>
         if (manager.getActionFor(ValidateElementRecursivelyAction.DEFAULT_ID) == null) {
             models.addAction(new ValidateElementRecursivelyAction(es, "Validate Models"));
         }
-        if (manager.getActionFor(ValidateElementDepthAction.DEFAULT_ID) == null) {
```

Comment:
This removal is not desired as the working plan is to address the performance issue on the server-side. This functionality is desired and, as far as I am aware, still working on models that are sufficiently small to not trigger the performance issue. A transparent fix on the server-side to accommodate larger models would allow this to work unchanged. Should the fix not be transparent, then we can re-implement it based on the new server spec once it is available.

Reply:
Withdrawn, as discussed at dev tag-up with user community reps.